docs/reference/standard-library.rst

```diff
@@ -38,40 +38,77 @@ Int
 Sequences
 ============
 
-Required for `pluscal procedures <procedure>`.
+Required for `pluscal procedures <procedure>`. A sequence is like a list in any other language. You write it like `<<a, b, c>>`, and the elements can be any other values (including other sequences).
 
 Seq(set)
-  The set of all sequences where the values are all elements of ``set``. |noenumerate| 
+  The set of all sequences where the values are all elements of ``set``. As the output is computed in a lazy way, this is not memory/cpu intensive. |noenumerate|
+
+  ::  
+
+  Seq({a, b}) = Seq({a, b})
+  \* in practice, the output is a set that contains the values:
+  \* {<<>>, <<a>>, <<b>>, <<a, b>>, <<b, a>>}
 
 Len(seq)
   Length of ``seq``.
 
+  ::
+
+    Len(<<1, 2, 3>>) = 3
```

Comment:
Minor preference: I'd instead do `Len(<<2, 4, 6>>) = 3`, so it's clear it's the length and not the last element. I know `Len` makes it clear but this is the kind of stuff you fret over when you do formal methods

Reply:
sounds good!